.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build Nehrim Plugin
+
+on:
+  push:
+    branches: master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: windows-2022
+    steps:
+      - name: Build Nehrim Plugin
+        uses: ModOrganizer2/build-with-mob-action@master
+        with:
+          mo2-third-parties: fmt gtest spdlog boost lz4
```

Comment:
Can be removed.

Reply:
What do you mean, exactly? I don't think you want me to remove the entire build step.

Reply:
Sorry, I meant the `mo2-third-parties` - I have updated the build action to recursively fetch third-party dependencies from the `mo2-dependencies` list. 

Basically, you only need `mo2-third-parties` when the third-party dependency is specific to the plugin being build (or missing from the list in the build action).

Reply:
Okay, I see. I felt like those were still valid so it seemed odd to be, but I understand now. I'll tweak that soon.